File: app/src/main/java/com/juniperphoton/myersplash/utils/NetworkUtil.java
package com.juniperphoton.myersplash.utils;


import android.content.Context;
import android.net.ConnectivityManager;
import android.net.NetworkInfo;

public class NetworkUtil {

    public static boolean usingWifi(Context context) {
        ConnectivityManager connectMgr = (ConnectivityManager) context.getSystemService(Context.CONNECTIVITY_SERVICE);
        NetworkInfo info = connectMgr.getActiveNetworkInfo();
        if (info != null) {
            return info.getType() == ConnectivityManager.TYPE_WIFI;
        } else return false;
    }
}

File: app/src/main/java/com/juniperphoton/myersplash/utils/NetworkUtil.kt
package com.juniperphoton.myersplash.utils

import android.content.Context
import android.net.ConnectivityManager

object NetworkUtil {
    fun usingWifi(ctx: Context): Boolean {
        var manager = ctx.getSystemService(Context.CONNECTIVITY_SERVICE) as ConnectivityManager
        var info = manager.activeNetworkInfo
        return info?.type == ConnectivityManager.TYPE_WIFI
    }
}